packages/account-v2/src/pages/LoginHistory/LoginHistory.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Table } from '@deriv-com/ui';
+
+const headers = ['Date and Time', 'Action', 'Browser', 'IP Address', 'Status'] as const;
+const columns = headers.map(header => ({
+    header,
+}));
+
+export const LoginHistory = () => {
+    return (
+        <div className='flex flex-col '>
+            <Table
+                columns={columns}
+                data={[
+                    {
+                        action: 'Login',
+                        browser: 'Chrome v122.0.0.0',
+                        datetime: '2024-03-07 12:04:42 GMT',
+                        ipAddress: '942',
+                        status: 'Successful',
+                    },
+                ]}
+                isFetching={true}
```

Comment:
what's the purpose of `isFetching` and is there a specific case we're setting it to `true`?

Reply:
some  variables and method that are required to run the table in deriv-ui are mentioned in the Docs making "isFetching" false has led the table to not load  as per observation 

<img width="1728" alt="Screenshot 2024-03-11 at 9 18 04 AM" src="https://github.com/binary-com/deriv-app/assets/158162395/5847ee54-61b1-4674-8f1b-094e98904487">

Reply:
Would you please upload a video referring to the scenario where the table is unable to render with this flag being false?